src/test/java/space/obminyashka/items_exchange/controller/UserControllerIntegrationTest.java

```diff
@@ -141,36 +140,28 @@ void addChild_badTotalAmount_shouldReturnBadRequest() throws Exception {
         assertTrue(Objects.requireNonNull(mvcResult.getResolvedException()).getMessage().contains(expectedErrorMessage));
     }
 
-    @Test
-    @WithMockUser(username = "admin")
-    void updateUserPassword_whenOldPasswordWrong_shouldThrowInvalidDtoException() throws Exception {
+    @ParameterizedTest
+    @WithMockUser(username = "user")
+    @MethodSource("listIncorrectDataPassword")
+    void updateUserPassword_whenDataIncorrect_shouldThrowIllegalArgumentException
```

Comment:
Don't move parameters to the next line by the bracket, just split them in the middle

Reply:
++